pkg/instancetype/instancetype.go

```diff
@@ -895,6 +896,16 @@ func applyMemory(field *k8sfield.Path, instancetypeSpec *instancetypev1alpha2.Vi
 		Guest: &instancetypeMemoryGuest,
 	}
 
+	// If memory overcommit has been requested, set the memory requests to be
+	// lower than the guest memory by the requested percent.
+	if instancetypeMemoryOvercommit := instancetypeSpec.Memory.OvercommitPercent; instancetypeMemoryOvercommit > 0 {
```

Comment:
`OvercommitPercent` needs some form of validation in the [instance type validation webhook](https://github.com/kubevirt/kubevirt/blob/e145467d7527c7f6b7e2b94e62d22992c410f15d/pkg/virt-api/webhooks/validating-webhook/admitters/instancetype-admitter.go) right?

Reply:
Added.